pages/common/tred.md

```diff
@@ -0,0 +1,13 @@
+# tred
+
+> Compute the transitive reduction of directed graphs.
+> Graphviz filters: `acyclic`, `bcomps`, `comps`, `edgepaint`, `gvcolor`, `gvpack`, `mingle`, `nop`, `sccmap`, `tred`, & `unflatten`.
+> More information: <https://www.graphviz.org/pdf/tred.1.pdf>.
+
+- Generate the transitive reduction of one or more directed graphs:
```

Comment:
@sbrl what do you think about `generate` here? I suggested that and before, it was `compute`. I think none of them really fit, but I can't think of something better.